Content.Server/Objectives/Commands.cs

```diff
@@ -0,0 +1,139 @@
+﻿#nullable enable
+using System.Linq;
+using Content.Server.Administration;
+using Content.Server.Players;
+using Content.Shared.Administration;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Objectives
+{
+    [AdminCommand(AdminFlags.Admin)]
+    public class AddObjectiveCommand : IClientCommand
+    {
+        public string Command => "addobjective";
+        public string Description => "Adds an objective to the player's mind.";
+        public string Help => "addobjective <username> <objectiveID>";
+        public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
+        {
+            if (args.Length != 2)
+            {
+                shell.SendText(player, "Expected exactly 2 arguments.");
+                return;
+            }
+
+            var mgr = IoCManager.Resolve<IPlayerManager>();
+            if (!mgr.TryGetPlayerDataByUsername(args[0], out var data))
+            {
+                shell.SendText(player, "Can't find the playerdata.");
+                return;
+            }
+
+
+            var mind = data.ContentData()?.Mind;
+            if (mind == null)
+            {
+                shell.SendText(player, "Can't find the mind.");
+                return;
+            }
+
+            if (!IoCManager.Resolve<IPrototypeManager>()
+                .TryIndex<ObjectivePrototype>(args[1], out var objectivePrototype))
+            {
+                shell.SendText(player, $"Can't find matching ObjectivePrototype {objectivePrototype}");
+                return;
+            }
+
+            if (!mind.TryAddObjective(objectivePrototype))
+            {
+                shell.SendText(player, "Objective requirements dont allow that objective to be added.");
+            }
+
+        }
+    }
+
+    [AdminCommand(AdminFlags.Admin)]
+    public class ListObjectivesCommand : IClientCommand
```

Comment:
Having one that lists all objectives in yaml would also be very convenient.

Reply:
neat idea, altough i think that will get unusable after ~20 objectives (which is not far off a basic traitormode implementation imo)